tests/phpunit/tests/plugins/test-wp-importer.php

```diff
@@ -116,4 +140,37 @@ public function test_simple_import() {
 		$this->assertEquals( 'fr', self::$model->term->get_language( $fr->term_id )->slug );
 		$this->assertEqualSetsWithIndex( array( 'en' => $en->term_id, 'fr' => $fr->term_id ), self::$model->term->get_translations( $en->term_id ) );
 	}
+
+	/**
+	 * This test checks the import of strings translations with already existing languages.
+	 * The strings translations are not the same between those of the existing languages and those of the import file.
+	 */
+	public function test_simple_import_with_existing_languages() {
```

Comment:
I believe the test is incomplete. I would expect that it checks that imported strings translations are merged with existing ones and do not overwrite them.

Reply:
Okay, see https://github.com/polylang/polylang/pull/1637/commits/a755bb50bbaa119f9dc0cec65b8930686aa82961.